docs/specs/tripgo.swagger.yaml

```diff
@@ -3328,6 +3331,9 @@ definitions:
 
   Trip:
     properties:
+      id:
+        type: string
+        description: Unique ID of the trip, which will change if the trip is saved.
```

Comment:
Why does this change when the trip is saved? Can't we keep it the same?

Reply:
done, will only change if the same id already exists in the database, which is unlikely